scripts/run_multi_model.py

```
@@ -0,0 +1,372 @@
+    import xml.etree.ElementTree as ET
+    try:
+        tree = ET.parse(junit_xml_path)
+        root = tree.getroot()
+        testsuites = root if root.tag == "testsuites" else [root]
+        tests = sum(int(ts.attrib.get("tests", 0)) for ts in testsuites)
+        failures = sum(int(ts.attrib.get("failures", 0)) for ts in testsuites)
+        errors = sum(int(ts.attrib.get("errors", 0)) for ts in testsuites)
+        skipped = sum(int(ts.attrib.get("skipped", 0)) for ts in testsuites)
+        failed = failures + errors
+        passed = tests - (failed + skipped)
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)